ArduCopter/config.h

```diff
@@ -59,6 +59,31 @@
 # define MAIN_LOOP_SECONDS 0.0025f
 # define MAIN_LOOP_MICROS  2500
 
+
```

Comment:
"Sorry, we could not display the entire diff because too many files (528) changed" from github.

Usually a sign there's something wrong with this commit. The branch should be **rebased** not merged.